lib/syscall_shim/include/uk/legacy_syscall.h

```diff
@@ -0,0 +1,27 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
```

Comment:
How did you create this list of system calls? Do they pertain to `aarch64`?

Reply:
Actually, the system calls added here are those that are not available on `aarch64`.

I choose the legacy syscalls based on the following criteria:
1. there is a newer syscall that includes their functionality
2. they are not architecture specific (i.e. they are not available only on x86, but on a wider set of architectures) [1]

For each of the system calls added here as legacy, I checked the man pages for alternatives:
 - all the syscalls provided by `lib/vfscore` that were included have a newer alternative that can provide their functionality (e.g. `open()` -> `openat()`, `access()` -> `faccessat() ` etc)
 - `fork()`, `getpgrp()` -> `clone()`, `getpgid()`
 - `time()`, `alarm()`, `pause()` -> `gettimeofday()`, `timer_settime()`, `sigsuspend()`
 - for `pause()`, see also [2]
 
 [1] https://marcin.juszkiewicz.com.pl/download/tables/syscalls.html
 [2] https://stackoverflow.com/questions/6328055/whats-the-problem-of-pause-at-all